vault/pkm.about.err.nie.md

```diff
@@ -0,0 +1,11 @@
+---
+id: 864cb973-0d0c-40a7-869b-efe6ad76798b
+title: NIY*
+desc: ''
+updated: 1622902900513
+created: 1622902629296
```

Comment:
this section still needs work and looks weird being by itself. can we add `published: false` in the frontmatter?